lib/gui/modules/analytics.js

```diff
@@ -39,6 +41,42 @@ const analytics = angular.module(MODULE_NAME, [
   require('../models/settings')
 ]);
 
+/**
+ * @summary Create an object copy with all absolute paths replaced with the path basename
+ * @function
+ * @private
+ *
+ * @param {Object} object - original object
+ * @returns {Object} transformed object
+ *
+ * @example
+ * const anonymized = hidePaths({
+ *   path1: '/home/john/rpi.img',
+ *   simpleProperty: null,
+ *   nested: {
+ *     path2: '/home/john/another-image.img',
+ *     path3: 'yet-another-image.img',
+ *     otherProperty: false
+ *   }
+ * });
```

Comment:
Ugh, good catch. I think we should provide a whitelist of base paths that are not anonymized. For example, we can whitelist everything that comes from `/dev`, etc. I'm not sure how `\\\\.\\PHYSICALDRIVE1` would react to this code, so better to unit test and make sure it doesn't get modified.